src/object/cli_obj.c

```diff
@@ -306,7 +306,10 @@ obj_layout_create(struct dc_object *obj, bool refresh)
 	int			rc;
 
 	pool = dc_hdl2pool(dc_cont_hdl2pool_hdl(obj->cob_coh));
-	D_ASSERT(pool != NULL);
+        if(pool == NULL) {
```

Comment:
(style)  code indent should use tabs where possible
(style)  suspect code indent for conditional statements (8, 12)
(style)  space required before the open parenthesis '('